docs/source/index.rst

```diff
@@ -18,6 +18,7 @@ Example
 -------
 
 .. image:: _static/img/example_issue.png
+.. _usage:
```

Comment:
This shouldn't be here; this makes the link below point up here, instead of to the `usage` page.